suite-common/formatters/src/formatters/tests/prepareCryptoAmountFormatter.test.ts

```diff
@@ -74,32 +73,6 @@ describe('CryptoAmountFormatter', () => {
                 }),
             ).toBe('0.00014898… ETH');
         });
-
```

Comment:
useless tests, they go against TS.

`CryptoAmountFormatter.format` does not let you call it with `symbol` other than `NetworkSymbol`, so unless someone overrides TS with `as` or `any`, this should never happen. And if someone does override TS, let's find out? :smiling_imp:  We don't want that in our codebase when we have utils like `isNetworkSymbol`.